plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -145,6 +146,14 @@ export class PersonState {
         if (propertiesHandled) {
             return person
         }
+        // This indicates an implementation problem on the user side and can cause lock contention in our
+        // ingestion pipeline.
+        if (person.version > MAX_PERSON_VERSION_ALLOWING_PROPERTIES_UPDATES) {
+            await captureIngestionWarning(this.db, this.teamId, 'too_many_person_properties_updates', {
+                personUuid: person.uuid,
+                eventUuid: this.event.uuid,
+            })
+        }
```

Comment:
I think this is missing an early `return` — wouldn't this just log the error and then process it anyway?